demo_nodes_cpp/src/parameters/list_parameters.cpp

```diff
@@ -27,7 +29,15 @@ int main(int argc, char ** argv)
   auto parameter_service = std::make_shared<rclcpp::parameter_service::ParameterService>(node);
 
   auto parameters_client = std::make_shared<rclcpp::parameter_client::SyncParametersClient>(node);
+  while (!parameters_client->wait_for_service(1s)) {
+    if (!rclcpp::ok()) {
+      std::cout << "Interrupted while waiting for the service. Exiting." << std::endl;
```

Comment:
I am not saying that the existing `cout`'s need to be updated in this PR (though since it would only take a couple of minutes I think it would be good to do).

But newly added code should not use it.